crates/api_models/src/payouts.rs

```diff
@@ -393,6 +394,54 @@ pub struct PayoutCreateResponse {
 
     /// The business profile that is associated with this payment
     pub profile_id: String,
+
+    /// Time when the payout was created
+    #[schema(example = "2022-09-10T10:11:12Z")]
+    #[serde(with = "common_utils::custom_serde::iso8601::option")]
+    pub created: Option<PrimitiveDateTime>,
+
+    /// List of attempts
+    #[schema(value_type = Option<Vec<PayoutAttemptResponse>>)]
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
lets remove this #[serde(skip_serializing_if = "Option::is_none")] since it will omit it from merchant response in case null

Reply:
We intend to remove the `attempts` field in case it's empty

Reply:
but why do we need to remove it?